libs/github/utils.js

```diff
@@ -76,26 +76,35 @@ async function handleRateLimit({ rateLimit, client }) {
  * @returns {Promise<{data: *, rateLimit: object}>}
  *
  * @example
- * let requestParams = { owner: 'gsa', repo: 'code-gov-integrations, state: 'open', labels: 'help wanted,code.gov', per_page: 10, page: 1};
+ * let requestParams = {
+ *   owner: 'gsa',
+ *   repo: 'code-gov-integrations,
```

Comment:
Should have a closing quote here on line 81? @froi

Reply:
And that's what happens when I don't run the linter 😞 

Greate catch.

Reply:
I didn't notice it was in the JSDoc. I was really confused as to how the tests and other things passed 🤣

Reply:
I will merge this PR and update the missing quote.